src/sequencing.js

```diff
@@ -0,0 +1,29 @@
+import { always, converge } from 'ramda';
+
+/* eslint-disable max-len */
+/**
+ * Accepts a list of side-effecting functions and returns a new function. When
+ * invoked, this new function is applied to some arguments, each side-effecting
+ * function is applied to those same arguments in the order in which they were
+ * declared. The return of each side-effecting function is discarded and the
+ * new function will always return `undefined`.
+ *
+ * @func sequencing
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
+ * @category Function
+ * @sig [((a, b, ...) -> x | undefined), ((a, b, ...) -> y | undefined), ...] -> (a -> b -> ... -> undefined)
+ * @param {Array} functions A list of side-effecting functions.
+ * @return {Function} A new function.
+ * @see {@link http://ramdajs.com/docs/#converge|converge}
+ * @example
+ *
+ * const log = message => console.log(message);
+ * const info = message => console.info(message);
+ *
+ * RA.sequencing([info, log])('foo'); //=> prints 'foo' via info then log
+ */
+/* eslint-enable max-len */
+const sequencing = converge(always(undefined));
```

Comment:
We have a stub undefined function.
```js
const seq = converge(stubUndefined);

export default seq;

Reply:
Anyway I am still struggling with the practicality of this combinator. Why does it always have to return undefined ? Why cannot it return at least first argument of the call ? The way it is implemented now makes it pretty hard to use it in any composition. I can use it in the following way, but in order for composition to work, the `seq` must be used as a last function in the composition.

```js
const showStudent = R.compose(
  seq(
    append('#student-info'),
    consoleLog
  ),
  csv,
  findStudent
);

Reply:
Yeah, agreed.

Reply:
Ok if agreed, I'am looking forward to the new signature then ;]

Reply:
I see a couple different return possibilities from the new function itself. With a rough equivalent of `fns => curryN(fns[0].length, (...args) => ...` (I'll be more defensive in actual implementation) we could:

1. Return the args, which is an array, back that the functions were applied to.
2. Return `nth(0)` of `args`.
3. Return `undefined` is args length is 0, `nth(0)` if args length is 1, and `identity` if args length is greater than 1.

Reply:
Let's go for `nth(0)` to make it consistent with `tap` (K combinator)

Reply:
Nice, should have something later today once I get a moment to myself. Thanks for the help!

Reply:
After looking at `tap`, it looks like it returns the first argument because the function must be urnary. `seq`, as it is currently defined, can accept any number of arguments and returning the first seems when there is more than one seems odd. Thoughts on if `seq` should only accept functions that are urnary OR rethinking/ensuring what are we are returning makes sense?

Reply:
Ok let me think about this before I answer. This is always the hardest part; making the util composable...

Reply:
Thought about it for some time. I'm for limiting the arity to 1, to make it predictable and easily composable. I can relate to authors of `tap` why they decided to the arity of 1. What you think ?

Reply:
Yeah, agreed.